cypress/integration/networking.spec.ts

```diff
@@ -0,0 +1,65 @@
+import type { OnboardedNode } from "../plugins/index";
+
+import * as commands from "../support/commands";
+import * as path from "path";
+import { ipcStub } from "../support";
+
+context("networking", () => {
+  it("replicates a project from one node to another", () => {
+    cy.task("startNode", 17000);
+    cy.task("onboardNode", 17000);
+
+    cy.task("startNode", 18000);
+    cy.task("onboardNode", 18000);
+
+    cy.task("connectNodes", [17000, 18000]);
+
+    cy.task<OnboardedNode>("withNode", 17000).then(node => {
```

Comment:
It looks to me, like this is all boilerplate that will be repeated for every test with two nodes. So how about encapsulating this in one function and do some like.

~~~js
withTwoNodes().then(([node1, node2]) => ...)
~~~

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/e46870ae78be463f7244faa979625f2a267cc92b